libutils/string_lib.h

```diff
@@ -190,6 +190,8 @@ char *StringVFormat(const char *fmt, va_list ap);
  */
 char *StringFormat(const char *fmt, ...) FUNC_ATTR_PRINTF(1, 2);
 
+long StringCopy(char *const dst, const char *const src, const long n);
```

Comment:
Some of the `const`s here do not make much sense for a header file/prototype, but not a big deal.